aws/resource_aws_chime_voice_connector.go

```diff
@@ -0,0 +1,132 @@
+package aws
+
+import (
+	"context"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/chime"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAwsChimeVoiceConnector() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAwsChimeVoiceConnectorCreate,
+		ReadContext:   resourceAwsChimeVoiceConnectorRead,
+		UpdateContext: resourceAwsChimeVoiceConnectorUpdate,
+		DeleteContext: resourceAwsChimeVoiceConnectorDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"outbound_host_name": {
```

Comment:
These should be sorted alphabetically to make them easier to locate. It is less important in a smaller resource like this, but in a larger one (like an ec2 instance), it can be useful

Reply:
Done!